library/src/fy_library/flows/create_template_model_using_parsed_fy_py_file/flow_template_model_fy.py

```diff
@@ -78,17 +77,5 @@ def __call__(self) -> FlowTemplateModel:
             return_type=parsed_flow_fy_py_file.return_type,
             properties=parsed_flow_fy_py_file.properties,
             methods=parsed_flow_fy_py_file.methods,
-            # TODO: change to ParsedAbstractPropertyFyPyFile after completed migration
-            property_setters=[
-                AbstractPropertyTemplateModel(
-                    python_class_name=PythonEntityName.from_pascal_case(""),
-                    abstract_property_name=cast(
-                        ParsedAbstractPropertyFyPyFile, parsed_fy_py_file
-                    ).abstract_property_name,
-                    property_type=cast(
-                        ParsedAbstractPropertyFyPyFile, parsed_fy_py_file
-                    ).property_type,
-                )
-                for parsed_fy_py_file in self._parsed_fy_py_files
-            ],
+            property_setters=self._parsed_fy_py_files,
```

Comment:
It is interesting that `property_setters: List[ParsedAbstractPropertyFyPyFile]` accept `List[ParsedFyPyFile]`. It might be that `mypy` does not check that level.